src/screens/Home/Home.tsx

```diff
@@ -102,16 +105,16 @@ function Home(props: HomeProps) {
           <div style={{ flex: 2 }}>
             <SiteMenu currentSite={currentSite} sites={allSites} />
             <Typography color='secondary'>
-              Current Period: {formatDateStringToLocal(getCurrentPeriod(), true)}
+              {intl(words.current_x, words.period)}: {formatDateStringToLocal(getCurrentPeriod(), true)}
             </Typography>
             <Typography color='secondary'>
-              Deadline: {formatDateStringToLocal(getNextPeriod(), true)}
+              {intl(words.next_deadline)}: {formatDateStringToLocal(getNextPeriod(), true)}
```

Comment:
I'm trying to be a bit more conservative when it comes to adding new translation labels due to our recent email exchanges with Sierra. Let me know if this (or any of my labels) strongly conflicts with the current Figma design. cc: @julianrkung